packages/store/src/StoreRead.sol

```diff
@@ -37,6 +73,13 @@ contract StoreRead is IStoreRead {
     return StoreCore.getRecord(tableId, keyTuple, fieldLayout);
   }
 
+  /**
+   * @notice Retrieves data for a specified field in a record.
+   * @param tableId The ID of the table.
```

Comment:
```suggestion
   * @notice Retrieves data for a specified field in a record.
   * @dev This overload loads the FieldLayout from storage. If the table's FieldLayout is known
   * to the caller, it should be passed in to the other overload to avoid the storage read.
   * @param tableId The ID of the table.
```